### PR TITLE
Downloads: add info on MacPorts

### PR DESCRIPTION
**Pull request by nilason**
Add information on availability and brief installation instructions with MacPorts.

### CODE REVIEW

File: source/site/forusers/alldownloads.rst [hunk 1]
```diff
@@ -543,6 +543,30 @@ QGIS nightly release
...
 
+MacPorts
+------------
+The package management system `MacPorts <https://www.macports.org>`_ offers both the latest release
+version (port ``qgis3``) and the long term version (port ``qgis3-ltr``). This will install QGIS with
+native architecture, Intel x86_64 or Apple ARM. Main software dependencies such as GDAL, PDAL and
+GRASS GIS are usually the latest version available.
+
+`Installing MacPorts and updating <https://guide.macports.org>`_ it and the *ports* are made with
+the *Terminal*. QGIS is however installed as an app bundle at ``/Applications/MacPorts/QGIS3.app``.
```
Comment (DelazJ) [inline source/site/forusers/alldownloads.rst]:
```suggestion
the *Terminal*. QGIS is however installed as an app bundle at :file:`/Applications/MacPorts/QGIS3.app`.
```

File: source/site/forusers/alldownloads.rst [hunk 2]
```diff
@@ -543,6 +543,30 @@ QGIS nightly release
...
+
+Get information of a port::
+
+  sudo port info qgis3
+
+Install port, e.g with GRASS GIS and OpenMP::
+
+  sudo port install qgis3 +grass +openmpi
```
Comment (DelazJ) [inline source/site/forusers/alldownloads.rst]: No idea what openmp (or openmpi?) is. Is it something relevant to QGIS install?

**Review by DelazJ — COMMENTED** [review]
Thanks for providing another way of instaal to macos users @nilason

File: source/site/forusers/alldownloads.rst (hunk 2 above)
Comment (nilason) [inline source/site/forusers/alldownloads.rst]:
```suggestion
Install port, e.g with GRASS GIS::

  sudo port install qgis3 +grass
```

File: source/site/forusers/alldownloads.rst (hunk 1 above)
Comment (nilason) [reply source/site/forusers/alldownloads.rst]: Good! Pushed this change.

File: source/site/forusers/alldownloads.rst (hunk 2 above)
Comment (nilason) [reply source/site/forusers/alldownloads.rst]: As it now is, it is "needed" for some of its dependents (if they use OpenMP). But I will see if there is an alternative way to solve that. I removed this part from the example.

**Pull request merged**